src/main/kotlin/graphics/scenery/backends/vulkan/VulkanNodeHelpers.kt

```diff
@@ -352,7 +379,14 @@ object VulkanNodeHelpers {
      *
      * Returns true if the node has been given a custom shader, and false if not.
      */
-    fun initializeCustomShadersForNode(device: VulkanDevice, node: Node, addInitializer: Boolean = true, renderpasses: Map<String, VulkanRenderpass>, lateResizeInitializers: MutableMap<Renderable, () -> Any>, buffers: VulkanRenderer.DefaultBuffers): Boolean {
+    fun initializeCustomShadersForNode(
+        device: VulkanDevice,
+        node: Node,
+        addInitializer: Boolean = true,
+        renderpasses: Map<String, VulkanRenderpass>,
+        lateResizeInitializers: MutableMap<Renderable, () -> Any>,
+        buffers: VulkanRenderer.DefaultBuffers)
```

Comment:
What's the reasoning behind handing the buffers over here? They could be determined from the `node`, not?

Reply:
':D I think I just did the line breaks there cause it was so long even at my screen. I did not add anything there

Reply:
And the VulkanRenderer.DefaultBuffers contain the scene ubos. The node does not know about them, so that should be the reason you included them there